test/inc/TestDef.h

```diff
@@ -17,6 +17,9 @@ static const std::wstring c_testFilePhaseEventName = L"ReunionTestFilePhaseEvent
 static const std::wstring c_testStartupPhaseEventName = L"ReunionTestStartupPhaseEventName";
 
 static const std::wstring c_testInstanceRedirectedPhaseEventName = L"ReunionTestInstanceRedirectedPhaseEventName";
+static const winrt::hstring c_rawNotificationPayload = L"<toast></toast>";
+static const IID c_comServerId = winrt::guid("ccd2ae3f-764f-4ae3-be45-9804761b28b2");
+static const IID c_fakeComServerId = winrt::guid("00000000-0000-0000-0000-000000000001");
```

Comment:
Don't use `static` on global variables in header files. This causes each client of the header file to get a separate copy of the variable. Use `constexpr` and/or `inline` to allow the variables to be shared.